ros2_odometry_node:
  ros__parameters:
    drive_type: "ACKERMANN"
    ackermann:
      wheelbase: 1.52439
      ticks_per_rev: 4096.0
      left_wheel_diameter: 0.623479
      right_wheel_diameter: 0.622806